webapp/views.py

```diff
@@ -716,3 +733,48 @@ def sitemap_index():
 
     response.headers["Content-Type"] = "application/xml"
     return response
+
+
+def marketo_describe():
```

Comment:
Where is this and `marketo_submit` used?

Reply:
Nowhere it was just handy to get information about what the rest API field names are. It can be deleted.